Library/Formula/llvm.rb

```diff
@@ -94,7 +94,7 @@ class Llvm < Formula
   deprecated_option "rtti" => "with-rtti"
   deprecated_option "disable-assertions" => "without-assertions"
 
-  if MacOS.version <= :snow_leopard
+  if MacOS.version <= :snow_leopard || build.with?("python")
```

Comment:
Is this not triggered automatically if you `brew install llvm --with-python`? That invokes the `depends_on :python` dep below, no?